scripts/apps/search/controllers/MultiActionBarController.js

```diff
@@ -87,4 +109,19 @@ export function MultiActionBarController(
         });
         return canPackage;
     };
+
+    /**
+     * @ngdoc method
+     * @name MultiActionBarController#canHighlightItems
+     * @description Checks if all items multi-selected are eligible to be highlighted
+     * @returns {Boolean}
+     */
+    this.canHighlightItems = function() {
+        var canHighlight = true;
+
+        multi.getItems().forEach((item) => {
+            canHighlight = canHighlight && authoring.itemActions(item).mark_item_for_highlight;
+        });
+        return canHighlight;
+    };
```

Comment:
Alternatively you could write this function like this:
`return multi.getItems().every(item => authoring.itemActions(item).mark_item_for_highlight);`